backup/setup/homeworks.json

```diff
@@ -1,1656 +1,1253 @@
-		"availableDate": {
-			"$date": "2017-09-05T12:00:00Z"
-		},
-		"dueDate": {
-			"$date": "2018-05-31T09:38:00Z"
-		},
-		"lessonId": {
-			"$oid": "59a3e4bba2049554a93fec94"
-		},
-		"courseId": {
-			"$oid": "59a3ce58a2049554a93fec6e"
-		},
-		"updatedAt": {
-			"$date": "2017-09-05T09:38:37.115Z"
-		},
-		"createdAt": {
-			"$date": "2017-09-05T09:38:37.114Z"
-		},
-		"__v": 0,
-		"publicSubmissions": false,
-		"private": false
+	"createdAt": {
+		"$date": "2020-11-05T07:08:50.055Z"
 	},
-	{
-		"_id": {
-			"$oid": "59ae711c1f513506904e1cc5"
-		},
-		"schoolId": {
-			"$oid": "5f2987e020834114b8efd6f6"
-		},
-		"teacherId": {
-			"$oid": "59ad4c412b442b7f81810285"
-		},
-		"name": "QUIZ: Vorbereitung für die nächste Stunde!",
-		"description": "<p>Quiz zu den Nachbarstaaten Deutschlands vorbereiten! (25 Kopien)</p>\r\n",
-		"availableDate": {
-			"$date": "2017-09-05T12:00:00Z"
-		},
-		"dueDate": {
-			"$date": "2018-05-31T12:00:00Z"
-		},
-		"private": true,
-		"lessonId": {
-			"$oid": "59a3e4bba2049554a93fec94"
-		},
-		"courseId": {
-			"$oid": "59a3ce58a2049554a93fec6e"
-		},
-		"updatedAt": {
-			"$date": "2017-09-05T09:40:44.117Z"
-		},
-		"createdAt": {
-			"$date": "2017-09-05T09:40:44.117Z"
-		},
-		"__v": 0,
-		"publicSubmissions": false
+	"updatedAt": {
+		"$date": "2020-11-05T07:08:50.055Z"
 	},
-	{
-		"_id": {
-			"$oid": "59afd42e1f513506904e1d1a"
-		},
-		"schoolId": {
-			"$oid": "5f2987e020834114b8efd6f6"
-		},
-		"teacherId": {
-			"$oid": "599ec1688e4e364ec18ff46e"
-		},
-		"name": "Bestandteile des menschlichen Auges",
-		"description": "<p>Vergleiche die <a href=\"https://schul-cloud.org/courses/59a3c657a2049554a93fec3a/topics/59a3c755a2049554a93fec52\" target=\"_blank\">Abbildung des Auges im Biologieunterricht</a>&#xA0;mit der nachfolgenden Abbildung und schreibe zu den Punkten jeweils die korrekte Bezeichnung auf:</p>\r\n\r\n<p><img alt=\"\" src=\"https://schul-cloud.org/files/file?file=courses/59a3c948a2049554a93fec5f/augeabb_task_web.png\"></p>\r\n\r\n<table border=\"1\" cellpadding=\"1\" cellspacing=\"1\" style=\"width:800px\">\r\n\t<tbody>\r\n\t\t<tr>\r\n\t\t\t<td>1):________________________</td>\r\n\t\t\t<td>2):________________________</td>\r\n\t\t</tr>\r\n\t\t<tr>\r\n\t\t\t<td>3):________________________</td>\r\n\t\t\t<td>4):________________________</td>\r\n\t\t</tr>\r\n\t</tbody>\r\n</table>\r\n\r\n<p>&#xA0;</p>\r\n\r\n<p>&#xA0;</p>\r\n",
-		"availableDate": {
-			"$date": "2017-09-06T10:55:00Z"
-		},
-		"dueDate": {
-			"$date": "2018-04-30T14:00:00Z"
-		},
-		"lessonId": {
-			"$oid": "59a529eba2049554a93fecff"
-		},
-		"courseId": {
-			"$oid": "59a3c948a2049554a93fec5f"
-		},
-		"updatedAt": {
-			"$date": "2017-09-06T10:55:42.299Z"
-		},
-		"createdAt": {
-			"$date": "2017-09-06T10:55:42.299Z"
-		},
-		"__v": 0,
-		"publicSubmissions": false,
-		"private": false
+	"__v": 0
+},
+{
+	"_id": {
+		"$oid": "5fa3a847a9c31a26f4d1d4a7"
```

Comment:
Homework with this id throws 504 gateway error when clicked. Same issue occurs with https://nbc-bc-372-improve-task-seed-data.cd.dbildungscloud.dev/homework/5fa3ac4ca9c31a26f4d1d55e.
Seed data seems to be faulty and therefore e2e tests are failing.